install/apm/node/install.yml

```diff
@@ -0,0 +1,26 @@
+id: node-agent
+name: Node Agent
+title: Node Agent Installation
+description: Start monitoring the performance of a Node application or service by installing the New Relic Node agent.
+
+target:
+  type: agent
+  destination: application
+  os:
+    - linux
+  context: |
+    ## WARNING
+    Using the command will automatically stop and attempt to restart all instrumentable applications, with all the same context, during the installation process. Once instrumented, redeploying any application(s) will require you to re-run the command. If you would like to manually install, please see our Node standard installation.
+
+    ## INFO
```

Comment:
Should we change the `INFO` and `WARNING` to `Info` and `Warning`?

Reply:
Everything else looks great, thanks @jbeveland27

Reply:
Yeah good callout. So currently I just copied what we display in the existing UI. I'd like to treat think this `context` field like markdown, and then the UI can parse and render that out in the sidebar. However there are some details that aren't fully figured out yet with limitations that might impose (such as preferred styling). I'll leave it like this for now, and we can revisit when we start to implement this in the UI.

![image](https://user-images.githubusercontent.com/6722433/132877770-889484d6-5076-4761-96ae-4f248a650df4.png)